picturae_importer.py

```diff
@@ -395,22 +463,25 @@ def generate_taxon_fields(self, index, taxon):
 
         author_insert = self.author
 
```

Comment:
improved assignment of rank id for genus level hybrids, previously would accidentally assign species rank ID to hybrids at the genus level.